### DISCUSSION TITLE
[Feature Request]: Shared PHP.ini

### DISCUSSION THREAD

**Discussion by iruzevic** [beyondcode/herd-community#347] [Ideas]
### Feature Description

It would be great if we could have a single shared php.ini file that is automatically added to every installed PHP version. This file would allow me to set default settings for all my PHP versions without the need to manually add them every time a minor PHP version is released.

**Comment by rinodrummer** [comment]
I think this feature should be considered as valuable because every time a new version of PHP is installed, the same settings have to be replicated rightly in the newly generated `php.ini`.

It could be placed in `%HerdLocation%/config/php` and injected via the `include_path` directive.

The logic should be the same behind the max upload size and the memory limit, but without exposing the user the input to manage it (or using a read-only input to copy the path).

The only issues could be:
- The use of the right separator (`:` on Unix-like or `;` on Windows) depending on the OS
- When a user wants to opt-out on the inject for a specific version (which could be solved by adding a new column on the available versions table name like "Include shared options" containing a checkbox on each row).
